topics/concourseCli/index.md

```diff
@@ -0,0 +1,10 @@
+---
+display_name: concourseCli
```

Comment:
The name of the project is just "Concourse CI". It does have a CLI but that's a separate repo, and it's called the `fly` CLI.
```suggestion
display_name: Concourse CI
```